forms-web-app/src/routes/validators/decision-date.js

```diff
@@ -0,0 +1,36 @@
+const { body } = require('express-validator');
+const moment = require('moment');
+
+const decisionDateCombiner = (value, { req }) => {
+  const day = req.body['decision-date-day'];
+  const month = req.body['decision-date-month'];
+  const year = req.body['decision-date-year'];
+
+  return `${year}-${month}-${day}`;
+};
+
+const deadlineDateValidator = (value) => {
+  const currentDate = moment();
+  const deadlineDate = value.add(12, 'weeks');
+
+  if (deadlineDate.isBefore(currentDate, 'days')) {
+    throw new Error('Deadline date has passed');
+  } else {
+    return value;
+  }
+};
+
+const rules = () => {
+  return [
+    body('decision-date-day').notEmpty(),
+    body('decision-date-month').notEmpty(),
+    body('decision-date-year').notEmpty(),
+    // body('decision-date-year').custom(decisionDateCombiner).isDate().custom(deadlineDateValidator),
```

Comment:
Can this be removed?

Reply:
Yup - removed. 

I had left this file alone as when I started this work, the decision date validation was in progress as a separate ticket and didn't want to cause conflicts. That said, I then renamed the file and extracted out some stuff - so I guess that went out of the window anyway. 

This file needs a refactor - can get rid of moment imo but it's outside the scope of this already overly large ticket